elastic/kibana
# [Workplace Search] Convert Sources pages to new page template (+ personal dashboard)

## Summary

Follow up to #102170 - this is the last PR for converting a chunk of Workplace Search views to the new KibanaPageTemplate. It's the largest and most complex one since Sources is already the most complex set of views, and requires accounting for the Personal Dashboard layout + updating that as well. As always, follow along by commit and disable whitespace diffs, etc.

## Screencaps

### Organization Sources

![organization_sources](https://user-images.githubusercontent.com/549407/122496063-ff9a8b00-cf9f-11eb-8e66-f5fb69b5129c.gif)

### Personal Dashboard

![private_sources](https://user-images.githubusercontent.com/549407/122496875-53f23a80-cfa1-11eb-9a8e-ebc66780e32a.gif)

(Not sure why this one spent so much time loading or had an error on the content sources page...maybe because I'd just added the slack connector?)

## QA

⚠️ **I need QA help on this one!** I'm fairly sure I QA'd everything, but it's strongly possible I missed something from just not knowing enough about Workplace Search. Please pull this down locally and check to make sure:

- [x] all pages look visually fine and load as expected
- [x] empty states look fine as well as filled states
- [x] document titles+breadcrumbs update as expected
- [x] reloading/refreshing the app on all pages works and doesn't cause strange behavior or crashes due to (e.g.) incorrect logic mount order

### Checklist
